### PR TITLE
feat(files): paste files from tobe_ui

### PR DESCRIPTION
**Pull request by sunxu42**
## Requirements

- [x] I reviewed my own code, you can now review the PR 😘

## Description

- [x] paste the files from ui project
- [x] set up the environments

## Screenshots
image

### CODE REVIEW

File: tobe_blog_web/vite.config.ts [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-import react from '@vitejs/plugin-react-swc'
+import react from '@vitejs/plugin-react-swc';
+import { defineConfig } from 'vite';
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  envDir: './env',
   plugins: [react()],
+  server: {
+    port: 8080,
```
Comment (LouisHongYi) [inline tobe_blog_web/vite.config.ts:9]: let's change to another port, not the same as the backend default port.

Comment (LouisYi-n) [inline tobe_blog_web/vite.config.ts:9]: Let's change to another port that is not the same as the backend default port.

**Review by LucienChenQima — APPROVED** [review]
LGTM

**Review by LucienChenXi92 — APPROVED** [review]

**Comment by LucienChenXi92** [comment]
Moving forwards, we need to remove the concept of "Domain" in the front-end and rename it as "Content"

**Pull request merged**